editor/src/components/canvas/multiplayer-cursors.tsx

```diff
@@ -89,8 +92,9 @@ export const MultiplayerPresence = React.memo(() => {
 
   return (
     <>
-      <MultiplayerCursors />
       <FollowingOverlay />
+      <MultiplayerShadows />
+      <MultiplayerCursors />
```

Comment:
file badly needs renaming

Reply:
or we have to move the other components to their own files

Reply:
it absolutely does, but I'd go with a separate PR to not pollute the diff